CHANGELOG.md

```diff
@@ -71,12 +52,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Add `@cumulus/ingest/providerClientUtils.buildProviderClient` function
   - Add `@cumulus/ingest/providerClientUtils.fetchTextFile` function
 
+- **CUMULUS-1731**
+  - Upgrade the version of the Thin Egress App deployed by Cumulus to v48
```

Comment:
This should probably go in the changed section

Reply:
Are you sure? I'd agree if this note was in isolation but it provides meaningful context for why 6 new input variables were added.

Reply:
I don't care that much. I was thinking both places since if someone looked for if TEA was upgraded they'd look at changed, but not that necessary.